src/create_disk.sh

```diff
@@ -147,14 +220,22 @@ case "$arch" in
         sgdisk -p "$disk"
         ;;
     s390x)
-        # NB: in the bare metal case when targeting ECKD DASD disks, this
-        # partition table is not what actually gets written to disk in the end:
-        # coreos-installer has code which transforms it into a DASD-compatible
-        # partition table and copies each partition individually bitwise.
-        sgdisk -Z $disk \
-        -U "${uninitialized_gpt_uuid}" \
-        -n ${BOOTPN}:0:+384M -c ${BOOTPN}:boot \
-        -n ${ROOTPN}:0:${rootfs_size} -c ${ROOTPN}:root -t ${ROOTPN}:0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        if [[ ${secure_execution} -eq 1 ]]; then
+            sgdisk -Z $disk \
+                -U "${uninitialized_gpt_uuid}" \
+                -n ${SDPART}:0:+200M -c ${SDPART}:se -t ${SDPART}:8DA63339-0007-60C0-C436-083AC8230908 \
```

Comment:
On [the Wikipedia page](https://en.wikipedia.org/wiki/GUID_Partition_Table), this GUID is marked "Reserved". Is there precedence for using this for our use case here? Otherwise, we probably should just use 0FC63DAF-8483-4772-8E79-3D69D8477DE4 instead.

Reply:
Yes, this is the partition type UUID (not the partition UUID). It's OK to have multiple partitions with the same type UUID.